src/commands/indexer/index.ts

```diff
@@ -257,6 +259,20 @@ export default class Indexer extends HealthCheck {
       }
     }
 
+    const v2BlockProcessingFilters =
```

Comment:
Might be nice to add a little comment about what this is doing and why for anyone that doesn't already have context